src/context.rs

```diff
@@ -1298,12 +1375,63 @@ impl Context {
         }
     }
 
-    /// Resets the value in a PCR register.
+    /// Resets the value in a PCR.
+    ///
+    /// # Arguments
+    /// *`pcr_handle` -  A [PcrHandle] to the PCR slot that is to be resetted.
```

Comment:
Is the `[PcrHandle]` part supposed to be a link? Or does it work like a link?

Reply:
In the documentation it becomes a link to that type.

Reply:
Damn, that's useful, would've been good to know about it some while ago